routes.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+  function call($controller, $action) {
+    require_once('controllers/' . $controller . '_controller.php');
```

Comment:
You're trusting raw user input here. Why is that bad and why should you avoid this?

Reply:
This is an safety issue? Like malicius user trying to discover my controllers and controller functions? I think i can pre set the availables urls and redirect the user for the corresponding controller and function. exemple: "/upcoming" -> new movies_controller->upcomming()